community_supplied/VendorSpecific/Paloalto/check-paloalto-processes-cpu-memory.rule

```diff
@@ -0,0 +1,314 @@
+/*
+ * Detects Panos processes (daemons) CPU and memory utilization threshold breaches and
```

Comment:
Can you please replace all instances to PANOS?

Reply:
Updated with required changes